packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/MultiFrameViewerContainer.spec.js

```diff
@@ -2,13 +2,14 @@ import { mount } from 'enzyme'
 import { Provider } from 'mobx-react'
 import sinon from 'sinon'
 import React from 'react'
+import asyncStates from '@zooniverse/async-states'
 
 import mockStore from '@test/mockStore'
-import { DraggableImage, MultiFrameViewerContainer } from './MultiFrameViewerContainer'
-import FrameCarousel from './FrameCarousel'
-import SingleImageViewer from '../SingleImageViewer/SingleImageViewer'
 import TranscriptionLineTool from '../../../../../../plugins/drawingTools/experimental/models/tools/TranscriptionLineTool/TranscriptionLineTool'
```

Comment:
```suggestion
import TranscriptionLineTool from '@plugins/drawingTools/experimental/models/tools/TranscriptionLineTool/TranscriptionLineTool'
```
The aliased path might make this easier to read.